pkg/components/util/install.go

```diff
@@ -26,10 +26,15 @@ import (
 
 	"github.com/kinvolk/lokomotive/pkg/components"
 	"github.com/kinvolk/lokomotive/pkg/k8sutil"
+	"github.com/kinvolk/lokomotive/pkg/signals"
 )
 
 // InstallComponent installs given component using given kubeconfig.
 func InstallComponent(name string, c components.Component, kubeconfig string) error {
+	// Get a new instance of SignalHandler and register to listen for os interrupts.
+	h := signals.NewSignalHandler()
+	h.Register()
```

Comment:
It seems we **will** register it multiple times. What if I interrupt the process after first component has been installed? Also, what's the point of interrupting, if it does not abort anything? I think we should print the correct error message, that the installing won't be interrupted if one hits CTRL+C.